nix/launcher-config.nix

```diff
@@ -60,6 +62,8 @@ let
     "--port" "8088"
     "--state-dir" dataDir.${os}
     "--genesis-block" "${dataDir.${os}}${dirSep}block0.bin"
+    "--sync-tolerance" "300s"
+    "--random-port"
```

Comment:
I'd prefer that we don't add --random-port for the selfnode. As it is mainly used for development, it is convenient using static ports such that utility scripts that interface directly with cardano-wallet continue to work without having to lookup the actual port.

It makes sense to add it to the other cluster modes though

Reply:
If you choose a fixed port, then it will fail to start if that port is in use.

Reply:
If self node is only for development, why are we producing installers?

Reply:
Removed `random-port`